lib/pagerduty.js

```diff
@@ -2,6 +2,15 @@
 const baseUrl = process.env.TEST ? 'http://localhost:3333/' : 'https://api.pagerduty.com/';
 const axios = require('axios');
 const utils = require('./utils');
+const axiosRetry = require('axios-retry').default;
+
+axiosRetry(axios, {
+  retries: 3,
+  retryDelay: axiosRetry.exponentialDelay,
+  retryCondition: (error) => {
+    return error.code === 'ETIMEDOUT';
```

Comment:
Hmmmm, I'm not sure axios will catch this. Can you try leaving the default retry settings? They should be for all generic kind of network issues